src/main/java/com/syntaxphoenix/spigot/smoothtimber/compatibility/coreprotect/CoreProtectChopListener.java

```diff
@@ -18,7 +19,9 @@ protected CoreProtectChopListener(final CoreCompat compat) {
     @EventHandler
     public void onChopEvent(final AsyncPlayerChoppedTreeEvent event) {
         for (final Location location : event.getBlockLocations()) {
+          SmoothTimber.getScheduler().runTask(location, () -> {
```

Comment:
Why is this done?

Reply:
u can refer to my first comment on why there’s nested schedules but all schedules called with a location passed to them are there because whatever is happening inside of them would throw an exception and not work unless done on the correct region thread
passing a location to universal scheduler schedules the runnable on the thread of that location